integration_tests/projects/014_broken_dbt_models/dbt_project.yml

```diff
@@ -0,0 +1,13 @@
+name: "fal_014"
+version: "1.0.0"
+config-version: 2
+profile: "fal_test"
+source-paths: ["models"]
+data-paths: ["data"]
+snapshot-paths: ["snapshots"]
+target-path: "{{ env_var('temp_dir') }}/target"
+vars:
+  fal-scripts-path: "scripts"
+
+models:
+  +schema: custom
```

Comment:
ATL: why are we setting this?

Reply:
No specific reason, this project was copied directly from a different one so we should probably clear all of them.

Reply:
I think this is tested in another project, but it's OK.